tests/testdata/version.txt

```diff
@@ -1 +1 @@
-v0.11.0-rc.1
+16f615403c89b2a49a656ae199cd2ba19e963f00
```

Comment:
Better to bump this to current `main` (`5f5ba8de0f04fc807a2c5683ab49235c8514cfc9`).

Either that, or (better) tag / release a new `v0.12.0-rc.0` of the contracts.

Reply:
Sequence of versioning would be:

1. Release new contracts version / tag. With the `babylon` submodule pointing to a babylon dev branch if necessary.
2. Release a `babylon-sdk` version / tag with that contract version.
3. Integrate that babylon sdk with babylon and other repos (integration, tests, etc.)
4. Adjust / update the babylon submodule ref in `babylon-contract` to `base/consumer-chain-support`.

Reply:
Yep we might need to start defining some release process, as well as cleaning the dependencies. 

My current thinking is to follow phase-2's procedure where Babylon remains the root of the dependency. The process is then like releasing Babylon node -> releasing every peripheral program.

This requires us to move out all the e2e tests for integration from Babylon to Babylon SDK.

Wdyt we create a github issue for the release procedure, and then address it after moving out those e2e tests from Babylon?